srcpkgs/musl/template

```diff
@@ -2,19 +2,17 @@
 pkgname=musl
 reverts="1.2.0_1"
 version=1.1.24
-revision=5
+revision=6
 archs="*-musl"
 bootstrap=yes
 build_style=gnu-configure
 configure_args="--prefix=/usr --disable-gcc-wrapper"
-makedepends="kernel-libc-headers"
 short_desc="Musl C library"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="http://www.musl-libc.org/"
 distfiles="http://www.musl-libc.org/releases/musl-${version}.tar.gz"
 checksum=1370c9a812b2cf2a7d92802510cca0058cc37e66a7bedd70051f0a34015022a3
-conflicts="glibc>=0"
```

Comment:
Why this?

Reply:
Because musl is only built for `*-musl` archs, and glibc is only built for `~*-musl` ones, so the conflict is totally unnecessary.